Personal_Accounts/src/test/java/org/openapitools/api_tests/aisp_accounts_accountId_transactions_get.feature

```diff
@@ -27,10 +27,10 @@ Scenario Outline: Get Account Transactions
     And match response.data.transaction[0] == '#object'
     And match response.links == '#object'
     And match response.links.self == '#string'
-    And match response.links.prev == '#string'
-    And match response.links.next == '#string'
-
+    # And match response.links.prev == '#string'
+    # And match response.links.next == '#string'
+    # "links.next" and  "links.prev" were not received in response but mentioned in api-spec
```

Comment:
The links `prev` and `next` are optional, so will not necessarily appear in the response